extra/genentries

```diff
@@ -1,11 +1,23 @@
 #!/bin/bash
 
-svgpath="svgicons"
+# this file generates an 'entries.dsv' for xlunch, which when 
+# generated, will contain lines like:
+#
+#  DeaDBeeF Music Player;/opt/deadbeef/deadbeef.png;/usr/bin/deadbeef
+
+# Usage:   genentries > /root/.config/xlunch/entries.dsv 
+
+# where to put the PNG icons we generate from svg
+svgpath="/usr/share/xlunch/svgicons"
```

Comment:
Why did you change this? The script is intended to drop the svgs into a local directory if no argument is given and some installation scripts rely on this behaviour.

Reply:
Not a biggie ... but why have them dotted around in local dirs when you can keep all the xlunch assets in /usr/share/xlunch (which is the best place for them)? .. 

And personally , I didn't want an `svgicons` dir in my $HOME dir, and any OS with many users would get many duplicate folders..

Reply:
Well the idea is that you can generate them when testing but without overriding your actual setup. You can also have multiple folders for different things. All of this could of course be achieved by passing the argument, but there are currently things  that depend on this which would break by this change.